base/dict.jl

```diff
@@ -6,6 +6,13 @@ const secret_table_token = :__c782dbf1cf4d6a2e5e3865d7e95634f2e09b5902__
 
 haskey(d::Associative, k) = in(k,keys(d))
 
+"""
+    getnull(d::Associative, key)
```

Comment:
if exported, this should also go in the rst docs - see CONTRIBUTING.md for full explanation, but you put the signature there then run `make docs` and the docstring content should get populated over.